### PR TITLE
Removed sphinx and added binder 

### LINKED ISSUE #5: Check out online rendering of notebooks (i.e. Binder)
It's important to make the notebooks easily executable online. It would be preferable to do it via [mybinder.org](https://mybinder.org/) unless there are performance issues (for example, if the notebooks take too long to render).

### PR DESCRIPTION
**Pull request by melissawm**
After discussions with the documentation team, we decided to make this repo simpler and remove sphinx, using only Binder for rendering notebooks.

This PR updates the configuration of the repo, removes unnecessary configuration files and includes gh-actions configurations to trigger rendering the notebooks (with binder) in issue/PR comments - [see more details here](https://mybinder.readthedocs.io/en/latest/howto/gh-actions-badges.html).

Closes #5 and #10.

### CODE REVIEW

File: .github/workflows/chatops-binder.yaml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#./github/workflows/chatops-binder.yaml
```
Comment (rgommers) [inline .github/workflows/chatops-binder.yaml:1]: did you mean to name this file `Chatops` and use that name in this file?

**Comment by rgommers** [comment]
Gave the button a quick try. Binder starts in the repo root. Can you make it start in `content/` instead? That's the only directory that users should be accessing right?

File: .github/workflows/chatops-binder.yaml (hunk 1 above)
Comment (melissawm) [reply .github/workflows/chatops-binder.yaml]: Fixed :)

**Comment by melissawm** [comment]
Hi @rgommers , just a ping to see if there's anything else left to do before we merge this :)

**Review by rgommers — APPROVED** [review]
LGTM, thanks Melissa!

**Comment by rgommers** [comment]
It's not happy yet, may be because it's still building, but need to check on this later:

image

**Pull request merged**